manifests/course/puppetize.pp

```diff
@@ -14,6 +14,12 @@
       mode  => '0644',
     }
 
+    # If we're operating in offline mode, start up the gitea service
+    # to host git repositories
+    if str2bool($offline) {
```

Comment:
This is already a boolean. In the ILT classes, it's validated, https://github.com/puppetlabs/pltraining-classroom/blob/master/manifests/init.pp#L35. We should refactor the virtuals to follow a similar pattern.